server/index.js

```diff
@@ -1,27 +1,34 @@
-'use strict';
+import loopback from 'loopback';
+import boot from 'loopback-boot';
 
-var loopback = require('loopback');
-var boot = require('loopback-boot');
-var debug = require('debug')('server');
+var app = loopback();
 
-var server = loopback();
-
-server.start = () => {
+app.start = function() {
```

Comment:
Why did you move away from `es6`? (Particularly curious when "converted to es6" is in the title of the PR!)